convert_hf_to_gguf.py

```diff
@@ -302,7 +302,7 @@ def prepare_tensors(self):
                 data_qtype: gguf.GGMLQuantizationType | bool = self.tensor_force_quant(name, new_name, bid, n_dims)
 
                 # Most of the codebase that takes in 1D tensors or norms only handles F32 tensors
-                if n_dims <= 1 or new_name.endswith("_norm.weight"):
+                if n_dims <= 1 or new_name.endswith("_norm.weight") or ".patch_embd.weight" in new_name:
```

Comment:
maybe it's better to add it to the list just right below, after `gguf.MODEL_TENSOR.V_ENC_EMBD_POS`